### PR TITLE
Add new option 'base_path'

### PR DESCRIPTION
**Pull request by timvink**

### CODE REVIEW

**Comment by fotrimzi** [comment]
Thanks. Will review/test this week.

File: docs/options.md [hunk 1]
```diff
@@ -11,19 +11,28 @@ You can customize the plugin by setting options in `mkdocs.yml`. For example:
...
-│   ├── basic_table.csv
+│   ├── tables/
+│   |   └── basic_table.csv
 │   └── index.md
 └── mkdocs.yml
 ```
 
+#### `base_path`
+
+Either `docs_dir` or `config_dir`. Default is `config_dir`. This is the base path where `mkdocs-table-reader-plugin` will search for input files.
```
Comment (fotrimzi) [inline docs/options.md:34]:
```suggestion
The base path where `mkdocs-table-reader-plugin` will search for input files.
The value is a string, one of `docs_dir` or `config_dir`. The default is `config_dir`. 
```

Comment (fotrimzi) [reply docs/options.md:34]: Just to clarify that the value for this param is one of the two choices listed.

**Comment by fotrimzi** [comment]
> Thanks. Will review/test this week.

It's great, works for my use case, and is more flexible than before. Thanks.

**Comment by timvink** [comment]
Thanks for the review, testing, and for this researching this option with me! Fun to collaborate.

**Comment by timvink** [comment]
Released: https://github.com/timvink/mkdocs-table-reader-plugin/releases/tag/v1.1.0

**Comment by fotrimzi** [comment]
> Thanks for the review, testing, and for this researching this option with me! Fun to collaborate.

Not at all, a pleasure, and thank _you_ for a great and useful plugin. Paul.

**Comment by timvink** [comment]
Unrelated, but I read your comment on https://github.com/fotrimzi/mkdocs2org

> It also needs a MkDocs plugin to create a single HTML file for the site, and rewrite the relative links into internal ones. So far, I’ve not found a way to do this. I have briefly looked at mkdocs-combine and mkdocs-with-pdf but haven’t got their intermediate one-page HTML to convert easily.

I wrote a plugin to do just that (combine all files into a single page). Never again, as there were so many edge cases to solve. Have you tried https://github.com/timvink/mkdocs-print-site-plugin ?

**Comment by fotrimzi** [comment]
I may have tried it (honestly, I can't remember) but I will give it another spin. Such plugins tend to be for printing a site, so I can imagine how messy it can get dealing with styling, pages, intermediate formats... I don't need all that, just a MkDocs site rendered as a single page. Sphinx-doc (my first experience with techdoc SSGs) has a build target for that, and I just naively assumed MkDocs would have the same. But I guess that's what plugins are for.
Anyway, thanks for the tip.

**Comment by fotrimzi** [comment]
https://github.com/fotrimzi/mkdocs-showcase/blob/main/docs/pdf.md#see-also

So I've previously stumbled over it but didn't look any further. I will now!

**Pull request merged**